src/Core/Grid/Definition/Factory/CurrencyGridDefinitionFactory.php

```diff
@@ -0,0 +1,214 @@
+                    'sortable' => false,
+                ])
+            )
+            ->add((new DataColumn('iso_code'))
+                ->setName($this->trans('ISO code', [], 'Admin.International.Feature'))
+                ->setOptions([
+                    'field' => 'iso_code',
+                ])
+            )
+            ->add((new DataColumn('conversion_rate'))
+                ->setName($this->trans('Exchange rate', [], 'Admin.International.Feature'))
+                ->setOptions([
+                    'field' => 'conversion_rate',
+                ])
+            )
+            ->add((new ToggleColumn('active'))
+                ->setName($this->trans('Enabled', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'active',
+                    'primary_field' => 'id_currency',
+                    'route' => 'admin_currencies_toggle_status',
+                    'route_param_id' => 'currencyId',
+                ])
+            )
+            ->add((new ActionColumn('actions'))
+                ->setName($this->trans('Actions', [], 'Admin.Global'))
+                ->setOptions([
+                    'actions' => (new RowActionCollection())
+                        ->add((new LinkRowAction('edit'))
+                            ->setIcon('edit')
+                            ->setOptions([
+                                'route' => 'admin_currencies_edit',
+                                'route_param_name' => 'currencyId',
+                                'route_param_field' => 'id_currency',
+                            ])
+                        )
+                        ->add((new SubmitRowAction('delete'))
+                            ->setName($this->trans('Delete', [], 'Admin.Actions'))
+                            ->setIcon('delete')
+                            ->setOptions([
+                                'method' => 'DELETE',
+                                'route' => 'admin_currencies_delete',
+                                'route_param_name' => 'currencyId',
+                                'route_param_field' => 'id_currency',
+                                'confirm_message' => $this->trans(
+                                    'Delete selected item?',
+                                    [],
+                                    'Admin.Notifications.Warning'
+                                ),
+                            ])
+                        ),
+                ])
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters()
+    {
+        return (new FilterCollection())
+            ->add((new Filter('iso_code', TextType::class))
+            ->setTypeOptions([
+                'required' => false,
+            ])
+                ->setAssociatedColumn('iso_code')
+            )
+            ->add((new Filter('active', ChoiceType::class)) //todo: change to YesNoChoiceType::class when available
```

Comment:
Do you plan to add the `YesNoChoiceType` in another PR ?

Reply:
it already there, you can use it @tomas862

Reply:
done :+1: